src/components/list-item/examples/list-item-checkbox.tsx

```
@@ -73,16 +72,21 @@ export class ListItemCheckboxExample {
     public render() {
         return (
             <Host>
-                <ul role="group" aria-label="Notification preferences">
+                <ul
+                    role="group"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<fieldset>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)